dspace-server-webapp/src/test/java/org/dspace/app/rest/signposting/controller/LinksetRestControllerIT.java

```diff
@@ -216,13 +259,18 @@ public void findOneItemJsonLinksetsWithBitstreams() throws Exception {
         context.restoreAuthSystemState();
 
         String url = configurationService.getProperty("dspace.ui.url");
-        getClient().perform(get("/signposting/linksets/" + item.getID() + "/json")
-                        .header("Accept", "application/linkset+json"))
+        String signpostingUrl = configurationService.getProperty("signposting.path");
+        String mimeType = "application/vnd.datacite.datacite+xml";
+        getClient().perform(get("/signposting/linksets/" + item.getID() + "/json"))
```

Comment:
please add an expectation about the response content header `application/linkset+json`

Reply:
fixed